### PR TITLE
Update microsoft-defender-endpoint-linux.md

### PR DESCRIPTION
**Pull request by mariusgx**
Fixed typos

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-linux.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ After you've enabled the service, you may need to configure your network or fire
 
 - Audit framework (`auditd`) must be enabled.
   >[!NOTE]
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-linux.md]:
```suggestion
  > [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor additional edit, please.

**Comment by yogkumgit** [comment]
@mariusgx Please implement the feedback, if you agree. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**